### PR TITLE
openstack: add periodic jobs for 4.14

### PR DESCRIPTION
**Pull request by EmilienM**

### CODE REVIEW

File: ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.14.yaml [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+  minimum_interval: 48h
+  steps:
+    cluster_profile: openstack-vh-mecha-az0
+    env:
+      ZONES_COUNT: "1"
+    workflow: openshift-e2e-openstack-proxy
+- as: e2e-openstack-csi-cinder
+  minimum_interval: 72h
+  steps:
+    cluster_profile: openstack-vh-mecha-central
```
Comment (MaysaMacedo) [inline ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.14.yaml:100]: Can you also add a skip of the tests listed here https://github.com/openshift/release/pull/37653?

Thanks

**Review by MaysaMacedo — COMMENTED** [review]
/lgtm cancel

**Comment by dulek** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-e2e-openstack-kuryr

File: ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.14.yaml (hunk 1 above)
Comment (EmilienM) [reply ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.14.yaml:100]: good point, done

**Comment by MaysaMacedo** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.14-e2e-openstack-csi-cinder

**Pull request merged**